packages/core/types/ts3.6/effects.d.ts

```diff
@@ -585,7 +585,7 @@ export function cps<Ctx, Fn extends (this: Ctx, ...args: any[]) => void>(
 
 export type CpsFunctionParameters<Fn extends (...args: any[]) => any> = Last<Parameters<Fn>> extends CpsCallback<any>
   ? AllButLast<Parameters<Fn>>
-  : never
+  : [never]
```

Comment:
This seems to fix the test issue. All tests are green. So unless a test is missing, this should be fine?